barretenberg/cpp/src/barretenberg/stdlib/honk_recursion/verifier/protogalaxy_recursive_verifier.test.cpp

```diff
@@ -17,27 +19,34 @@
 namespace bb::stdlib::recursion::honk {
 template <typename RecursiveFlavor> class ProtoGalaxyRecursiveTests : public testing::Test {
   public:
```

Comment:
This will look like a lot of diffs but I actually just went back to the Inner-Outer pattern which is what makes sense and makes it possible to instantiate the tests with the simulator (i.e. we still fold actual instances but the recursive folding/decider verifier is instantiated with the simulator);.